Mysql Inventory Bridge/src/main/java/net/craftersland/bridge/inventory/objects/BlackListedItem.java

```diff
@@ -0,0 +1,63 @@
+package net.craftersland.bridge.inventory.objects;
+
+import org.bukkit.Material;
+import org.bukkit.inventory.ItemStack;
+
+import java.util.Objects;
+
+public class BlackListedItem {
+
+    private Material material;
+    private int customDataId;
+
+    public BlackListedItem(ItemStack itemStack) {
+        if (itemStack != null && itemStack.getItemMeta() != null) {
+            this.material = itemStack.getType();
+            if (itemStack.getItemMeta().hasCustomModelData()) {
+                this.customDataId = itemStack.getItemMeta().getCustomModelData();
+            }
+        }
+    }
+
+    public BlackListedItem load(String itemRepresentation) {
```

Comment:
I couldn't find where you are using this method.

Reply:
I'm not using it, it was from the blacklist system using the config. May be helpful in the future.